product_packaging_usability/models/product_packaging.py

```diff
@@ -0,0 +1,17 @@
+# Copyright Akretion (http://www.akretion.com/)
+
+from odoo import models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    def ui_goto_packaging_view(self):
+        self.ensure_one()
+        action = self.env.ref("product.action_packaging_view")._get_action_dict()
```

Comment:
_for_xml_id() is better (check in code)

Reply:
Not sure to understand, you say second is better ?
```
self.env.ref("product.product_product_6")

self.env["product.product"]._for_xml_id("product.product_product_6")
```

I think first is better

`self.env.ref` is used in whole odoo src

Reply:
```
action = self.env.ref("product.action_packaging_view")._get_action_dict()

action = self.env["ir.actions.act_window"]._for_xml_id(
            "product.action_packaging_view"
        )

```

I prefer first but ok